reduxpractice/src/actions/index.js

```diff
@@ -0,0 +1,22 @@
+ export const add=(data)=>
+{
+   return { type:"ADD",payload:data }     
+}
+
+export const loading=(data)=>
+{
+    return {type:"loading",payload:data}
```

Comment:
if you are returning only one line you could do this 
 export const add=(data)=>({ type:"ADD",payload:data });